apps/roam/src/components/DiscourseFloatingMenu.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import { Popover, Menu, MenuItem, Button, Position } from "@blueprintjs/core";
+
+export const DiscourseFloatingMenu = () => (
+  <div id="discourse-floating-menu" className="bottom-right dark">
+    <Popover
```

Comment:
Let's try to move the popup so it is centered and above the button.  Here's what I currently see
<img width="207" height="128" alt="image" src="https://github.com/user-attachments/assets/882148d2-5f4b-4aa2-b908-804912fe3286" />

Usually this will be done by adjusting the `css` of the wrapper (`floatingMenuAnchor`) or the target (`<Button>`) and maybe adding modifiers to the `<Popover>`

You can see Sid's Node Tag implementation for reference:
https://github.com/DiscourseGraphs/discourse-graph/pull/304/files